src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorCSharpLanguageMiddleware.ts

```diff
@@ -69,12 +69,19 @@ export class RazorCSharpLanguageMiddleware implements LanguageMiddleware {
                         continue;
                     }
                 } else {
-                    this.logger.logVerbose(
-                        `Re-mapping text ${edit.newText} at ${edit.range} in ${uri.path} to ${remappedResponse.range} in ${documentUri.path}`);
+                    const remappedEdit = new vscode.TextEdit(remappedResponse.range, edit.newText);
+                    const [codeActionUri, codeActionEdit] = this.tryApplyingCodeActions(uri, remappedEdit);
```

Comment:
Interesting. I don't remember why we call `tryApplyCodeActions` when we are unable to remap edits (in the if block above). Looks like in this case, we remap successfully and then we call `tryApplyCodeActions`. @ryanbrandenburg @NTaylorMullen, do you recall why this was designed this way? This all seems too complicated.

Reply:
I agree, especially since the FullyQualifiedNamespace translator seems to be the only composite code action translator in use. I kept the original logic in tact (calling `tryApplyCodeActions` when we are unable to remap edits), in case there were some legacy reasons / code paths which could execute it. I added this call to `tryApplyCodeActions` on the main path so the FullyQualifiedNamespace translator can apply the necessary edits when required.